index.html

```diff
@@ -23,19 +23,167 @@
 
       <!-- Headline -->
 
-      <section id="headline" class="flex-col">
+      <section id="headline" class="flex-col about">
+          <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
 
-        <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
+          <p class="flex-row blue-light">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
-        <p class="flex-row blue-light">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <p class="flex-row p-medium purple-light">LET’S CONNECT</p>
+
+          <!-- Social icons -->
+          <ul class="flex-row social">
+            <li>
+              <a href="https://twitter.com/AkashaRojee" target="_blank" rel="noopener noreferrer">
+                <img src="images/social-icons/twitter.png" alt="Twitter icon">
+              </a>
+            </li>
+            <li>
+              <a href="https://linkedin.com/in/AkashaRojee" target="_blank" rel="noopener noreferrer">
+                <img src="images/social-icons/linkedin.png" alt="Linkedin icon">
+              </a>
+            </li>
+            <li>
+              <a href="https://medium.com/@Akasha.Rojee" target="_blank" rel="noopener noreferrer">
+                <img src="images/social-icons/medium.png" alt="Medium icon">
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/AkashaRojee" target="_blank" rel="noopener noreferrer">
+                <img src="images/social-icons/github.png" alt="GitHub icon">
+              </a>
+            </li>
+            <li>
+              <a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+                <img src="images/social-icons/angellist.png" alt="AngelList icon">
+              </a>
+            </li>
+          </ul>
+      </section>
+
+      <!-- Cards container -->
+
+      <section id="works" class="grid">
+
+        <!-- First card -->
+
+        <div class="work-card flex-col">
+
+          <img class="flex-row" src="/images/works/work1.png" alt="Screenshot of work1">
+
+          <div class="work-info">
+            <span class="title flex-row blue-main">Tonic</span>
+            <div class="subtitle flex-row">
+              <div class="blue-light">CANOPY</div>
+              <div class="grey-main">Back-End Dev</div>
+              <div class="grey-main">2015</div>
```

Comment:
There should be some space between those devs
![wrong](https://user-images.githubusercontent.com/58376020/120894564-63749980-c619-11eb-9fb7-31741c2c66ab.png)

![right](https://user-images.githubusercontent.com/58376020/120894574-6e2f2e80-c619-11eb-947a-ced5b3111799.png)

Reply:
I have now added an element to serve as the circle separator between the text items.